cmd/paratime/denomination/set.go

```diff
@@ -0,0 +1,69 @@
+package denomination
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
+
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+var (
+	symbol string
+
+	setDenomCmd = &cobra.Command{
+		Use:   "set <network> <paratime> <denomination> <number_of_decimals>",
+		Short: "Set denomination",
+		Args:  cobra.ExactArgs(4),
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			networkArg, ptArg, denomArg, decimalsArg := args[0], args[1], args[2], args[3]
+
+			if symbol == "" {
+				symbol = denomArg
+			}
+
+			if symbol == config.NativeDenominationKey {
+				cobra.CheckErr(fmt.Errorf("use denom set-native cmd for setting paratime's native denomination"))
```

Comment:
Hm is there a reason why this is forbidden?

Reply:
Not really, maybe someone could set native denom without knowing that _ is native denom key. I'll remove this.